icepyx/tests/test_variables.py

```diff
@@ -1,6 +1,27 @@
+import json
+
+import pytest
+
+from icepyx.core.variables import Variables
+from icepyx.core.query import Query
 import icepyx.core.variables as variables
 
 
+@pytest.fixture()
+def variables_obj_atl06():
+    return Variables(
+        "file", 
```

Comment:
Is it possible to use `vartype="order"` instead of `vartype="file"` to dynamically download the file?